### ISSUE TITLE
Inconsistent spacing between file properties and username with long view

### ISSUE THREAD

**Issue by causality-loop** [solidiquis/erdtree#179]
Awesome package, thank you.

When running with the `--long` flag, sometimes I notice a space of 2-3 characters between the properties (`.rw-r--r--`) and username (anticipated behavior, nice output).

However, upon running something like `erd -l` again, the spacing between the properties and user might change to 10 characters, then the next time revert to normal, then the next time change to 30 characters, etc.

I'm running Fedora 38.  The above behavior is observed in Kitty, Alacritty, and Terminal.  I installed via Brew and also tried latest GH release (erd-v3.0.0-aarch64-unknown-linux-gnu.tar.gz).

**Comment by solidiquis** [comment]
Interesting. Would you happen to have a screenshot?

**Comment by causality-loop** [comment]
Here you go:

![Screenshot from 2023-05-26 08-18-43](https://github.com/solidiquis/erdtree/assets/77332903/c8d33c66-bde3-4b13-9af5-18d5ee68941a)

**Comment by solidiquis** [comment]
That is so strange.. I'm doing the same thing on Alacritty and can't seem to replicate this issue. This is happening consistently for you across Kitty, Alacritty, and Terminal? Are you running TMUX by change?

<img width="939" alt="Screen Shot 2023-05-25 at 6 22 34 PM" src="https://github.com/solidiquis/erdtree/assets/45523555/beff99f9-3210-4ff0-81a6-f759e500643d">

**Comment by causality-loop** [comment]
Yes, the behavior is observed in all 3 terminals and I'm not running Tmux.  Let me try again with no config file....yup, when I remove `kitty.conf`, it still happens.

**Comment by solidiquis** [comment]
Do you still happen to get the same issue when running this with `--color none` ?

**Comment by causality-loop** [comment]
I noted some weird behavior.  I just ran `erd -l --color none` in a bunch of directories.  One thing I noticed was that one time I ran it, my username was replaced with `colordx995:994`.  Unfortunately, using `--color none` didn't help, though.

Another interesting thing that I noticed was that the issue only seems to happen in directories that contain directories.  When I run `erd -l` in a directory containing only files, the output is fine.

**Comment by solidiquis** [comment]
My last question is before I try to replicate this.. did you happen to edit your `LS_COLORS` environment variable?

**Comment by causality-loop** [comment]
Good idea!  I did not do that, but let me try running again with the default bashrc (on Fedora, `/etc/bashrc`)...... No change :(

**Comment by solidiquis** [comment]
Listen—I'm doing this as a labor of love with whatever free-time I can manage to scrape as someone with a full-time job and a father who is in critical condition from Type II diabetes, Hep C, and chronic kidney failure.

I'm sorry that I'm not able to address this issue at a velocity that pleases you but that's just what you get with FOSS sometimes; but the beauty of FOSS is that you have just as much power to help fix the issue as well as someone who uses Fedora—that would actually be very much appreciated. Otherwise please don't go making such claims.
